instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -394,13 +406,21 @@ def _prepare(
 
 
 def _on_finish(tracer, server_histograms, func, handler, args, kwargs):
-    response = func(*args, **kwargs)
-
-    _record_on_finish_metrics(server_histograms, handler)
+    try:
```

Comment:
Not sure we should catch user code exception and change code the behavior, or if we do we should re-raise later.

Reply:
Its just a finally block to ensure that we finish the trace span and metrics, even if the user code raises an exception.  It doesn't catch, so it shouldn't need to re-raise?

Reply:
Ah, overlooked that. Sorry for the noise!